src/main/resources/openAPI/openapi-generator-maven-plugin/templates/JavaSpring/pojo.mustache

```diff
@@ -4,6 +4,7 @@
 @ApiModel(description = "{{{description}}}"){{/description}}
 {{>generatedAnnotation}}{{#discriminator}}{{>typeInfoAnnotation}}{{/discriminator}}{{>xmlAnnotation}}
 @JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
Eg forstår ikkje alt dette, men det er ok. Er JsonInclude (fjerning av null-verdier) ein føresetnad for månadleg oppdatering? Eller er det to uavhengige ting?

Reply:
Hver commit her løser to helt uavhengige issues